freeipa/src/main/resources/freeipa-salt/salt/monitoring/scripts/cipa.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+import os
+import subprocess
+import shutil
+import sys
+
+def run_cipa():
+    proc = subprocess.Popen("/usr/local/bin/cipa --output metrics",
+                            stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE,
+                            stdin=subprocess.PIPE,
+                            shell=True)
+    output, err = proc.communicate()
+    return proc.returncode, output, err
+
+def check_content(fname):
+    with open(fname) as dataf:
+        return any("--output" in line for line in dataf)
+
+def main():
+    main_patched_script="/usr/local/lib/python3.6/site-packages/checkipaconsistency/main.py"
+    if not os.path.exists(main_patched_script) or not check_content(main_patched_script):
+        print("No cipa tool installed or OpenMetrics format output is not supported.")
+        sys.exit(0)
+    if not os.path.exists("/var/run/salt-master.pid") or not os.path.exists("/usr/local/bin/cipa") or not os.path.exists("/root/.config/checkipaconsistency"):
```

Comment:
`not os.path.exists("/var/run/salt-master.pid")` I don't think it's necessary, all freeipa instances are salt masters